doc/classes/PhysicalBone3D.xml

```diff
@@ -74,6 +74,9 @@
 		<member name="gravity_scale" type="float" setter="set_gravity_scale" getter="get_gravity_scale" default="1.0">
 			This is multiplied by the global 3D gravity setting found in [b]Project &gt; Project Settings &gt; Physics &gt; 3d[/b] to produce the body's gravity. For example, a value of 1 will be normal gravity, 2 will apply double gravity, and 0.5 will apply half gravity to this object.
 		</member>
+		<member name="joint_exclude_nodes_from_collision" type="bool" setter="set_joint_exclude_nodes_from_collision" getter="get_joint_exclude_nodes_from_collision" default="true">
+			If [code]true[/code], the two bodies of the nodes are not able to collide with each other.
```

Comment:
This description, while copied from `Joint`, is not very clear, what two nodes and bodies?

From a glance at the code it appears that `PhysicalBone3D` has a joint where the first body is the parent bone and the second body is this bone

So some more specific description would be good

Reply:
Assuming my assumptions about the way it works that would be good IMO

Reply:
Done!